src/utils/openmc/material.py

```diff
@@ -76,7 +76,9 @@ def set_id(self, material_id=None):
 
         # If the Material already has an ID, remove it from global list
         if not self._id is None:
-            MATERIAL_IDS.remove(self._id)
+            try:
```

Comment:
Ok, if I understand it correctly, your modification allows the user to set the ``_id`` attribute without using the ``set_id(...)`` routine. In this scenario, the global ``MATERIAL_IDS`` list will not contain the ``Material's`` id, hence the need for the ``try-catch``. Or is there some other scenario you are trying to accommodate?